integration/kubernetes/k8s-sysctls.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	pod_name="sysctl-test"
+	get_pod_config_dir
+}
+
+@test "Setting sysctl" {
+	# Create pod
+	kubectl apply -f "${pod_config_dir}/pod-sysctl.yaml"
+
+	# Check pod creation
+	kubectl wait --for=condition=Ready pod "$pod_name"
+
+	# Check sysctl configuration
+	cmd="cat /proc/sys/kernel/shm_rmid_forced"
+	kubectl exec $pod_name -- sh -c "$cmd" | grep 0
```

Comment:
This looks correct, but why not anchor to guarantee we're getting what we expect? For example, what if -- somehow -- this file contained the value `10`?

Maybe something like:

```sh
result=$(kubectl exec $pod_name -- sh -c "$cmd")
[ "${result}" = 0 ]
```

Reply:
@jodh-intel changes applied